api/namex/services/lookup/name_request_filing_actions.py

```diff
@@ -63,8 +63,8 @@ def teardown(self, exception):
         ('CLC', 'LLC', 'Extraprovincial Limited Liability Company', 'Extraprovincial Change of Name', 'static', None, 'https://www2.gov.bc.ca/assets/gov/employment-business-and-economic-development/business-management/permits-licences-and-registration/registries-forms/form_37_xco_-_name_change.pdf', None),
         ('RLC', 'LLC', 'Extraprovincial Limited Liability Company', 'Extraprovincial Reinstatement', 'static', None, 'https://www2.gov.bc.ca/gov/content/employment-business/business/managing-a-business/permits-licences/businesses-incorporated-companies/forms-corporate-registry#information-packages', None),
         ('AL', 'LLC', 'Extraprovincial Limited Liability Company', 'Extraprovincial Assumed Name', 'static', None, 'https://www2.gov.bc.ca/assets/gov/employment-business-and-economic-development/business-management/permits-licences-and-registration/registries-forms/form_37_xco_-_name_change.pdf', None),
-        ('FR', 'GP', 'General Partnership/Sole Proprietorship/DBA', 'Registration', 'onestop', None, None, None),
-        ('CFR', 'GP', 'General Partnership/Sole Proprietorship/DBA', 'Change of Name', 'onestop', None, None, None),
+        ('FR', 'GP', 'General Partnership/Sole Proprietorship/DBA', 'Registration', 'lear', None, None, None),
```

Comment:
@thorwolpert we might need to change this mapping to support the new design of SP/GP registration. As per the design the lear should be able to identify whether its a General Partnership or Sole Proprietorship or DBA. While making the change here we need to change `request_type_mapping` for the same

Reply:
No, that value is used by the auth dashboard to know where to route to.
It should be ok to change the value here, as the auth ui will pick it up.